hotel/__manifest__.py

```diff
@@ -8,7 +8,7 @@
                Services Pvt. Ltd., OpenERP SA",
     "category": "Hotel Management",
     "website": "https://github.com/OCA/vertical-hotel",
-    "depends": ["sale_stock", "account"],
+    "depends": ["sale_stock", "account", "uom"],
```

Comment:
@pavan-serpentcs Any good reason to add "uom" in dependency?

The stock modules already have dependency products, and the products have the uom module as a dependency.